weed/filer/filer_deletion.go

```diff
@@ -94,23 +94,23 @@ func (f *Filer) doDeleteFileIds(fileIds []string) {
 }
 
 func (f *Filer) DirectDeleteChunks(chunks []*filer_pb.FileChunk) {
-	var fildIdsToDelete []string
+	var filledIdsToDelete []string
```

Comment:
should have been `fileIdsToDelete`

Reply:
Easier to delete this and re-submit instead of rebasing.